### PR TITLE
Form view

### PR DESCRIPTION
**Pull request by thaomonster**
Hi, here are my part of the code for the form view-page.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,40 +1,79 @@
...
+  homePage.classList.add('hidden')
+};
+
+function homeButtonOnClick() {
+  displayHomePage();
+  homeHideButton();
+  homeAddButtons();
+};
+
+function displayHomePage() {
```
Comment (peytonjo) [inline src/main.js:54]: this functions name suggest that it is displaying the home page but the function holds the status of random cover and save cover buttons

File: src/main.js [hunk 2]
```diff
@@ -1,40 +1,79 @@
...
+  randomCoverButton.classList.remove('hidden')
+  saveCoverButton.classList.remove('hidden')
+};
+
+function homeHideButton() {
+  homeButton.classList.add('hidden')
+};
+
+function homeAddButtons() {
```
Comment (peytonjo) [inline src/main.js:63]: this suggests that it is adding buttons but the function only changes the status of the home page and cover form.

File: src/main.js (hunk 1 above)
Comment (thaomonster) [reply src/main.js:54]: Fixed!

File: src/main.js (hunk 2 above)
Comment (thaomonster) [reply src/main.js:63]: Fixed and pushed to Github!

**Pull request merged**